src/rabbit_vhost_sup_sup.erl

```diff
@@ -53,7 +53,7 @@ init([]) ->
     %% unless the operator opts in.
     RestartStrategy = vhost_restart_strategy(),
     ets:new(?MODULE, [named_table, public, {keypos, #vhost_sup.vhost}]),
-
+    %% TODO massive hack to start the fifo metrics while don't get fixed in ra
```

Comment:
Was this TODO addressed? Should it be addressed?

Reply:
It was addressed, but the comment was left behind. Fixing it now.